plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/impl/sql/StandartSQLDialectQueryGenerator.java

```diff
@@ -0,0 +1,386 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2022 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.model.impl.sql;
+
+import org.jkiss.code.NotNull;
+import org.jkiss.code.Nullable;
+import org.jkiss.dbeaver.Log;
+import org.jkiss.dbeaver.model.DBPAttributeReferencePurpose;
+import org.jkiss.dbeaver.model.DBPDataSource;
+import org.jkiss.dbeaver.model.DBPEvaluationContext;
+import org.jkiss.dbeaver.model.DBUtils;
+import org.jkiss.dbeaver.model.data.*;
+import org.jkiss.dbeaver.model.exec.DBCLogicalOperator;
+import org.jkiss.dbeaver.model.sql.SQLUtils;
+import org.jkiss.dbeaver.model.struct.DBSAttributeBase;
+import org.jkiss.dbeaver.model.struct.DBSContextBoundAttribute;
+import org.jkiss.utils.CommonUtils;
+import org.jkiss.utils.Pair;
+
+import java.lang.reflect.Array;
+import java.util.List;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class StandartSQLDialectQueryGenerator implements SQLDialectQueryGenerator {
+    private Log log = Log.getLog(StandartSQLDialectQueryGenerator.class);
+
+    public static StandartSQLDialectQueryGenerator INSTANCE = new StandartSQLDialectQueryGenerator();
+
+    public static final Pattern PATTERN_COLUMN_NAME = Pattern.compile(
+        "(([a-z_][a-z0-9_]*)|(\\\"([a-z_][a-z0-9_]*)\\\"))(\\.(([a-z_][a-z0-9_]*)|(\\\"([a-z_][a-z0-9_]*)\\\")))*",
+        Pattern.CASE_INSENSITIVE
+    );
+
+
+    @Override
+    public void appendQueryConditions(DBPDataSource dataSource,
+        @NotNull StringBuilder query,
+        @Nullable String tableAlias,
+        @Nullable DBDDataFilter dataFilter) {
```

Comment:
here and in other places too..
```suggestion
        @Nullable DBDDataFilter dataFilter
    ) {
```

We decided to use such formatting, please, use this pattern.